multimodal/health_multimodal/image/model/modules.py

```diff
@@ -0,0 +1,85 @@
+#  -------------------------------------------------------------------------------------------
```

Comment:
Perhaps `layers.py` is a better name for this file, or if we want something more specific `mlp.py`?

Reply:
https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module
https://pytorch.org/docs/stable/generated/torch.nn.Module.html

Reply:
Just following the torch convention.